src/UIAutomation/Microsoft.UI.UIAutomation/Standins.cpp

```diff
@@ -545,6 +564,13 @@ namespace winrt::Microsoft::UI::UIAutomation::implementation
         return result;
     }
 
+    // AutomationRemoteConnectionBoundObject
+
+    AutomationRemoteConnectionBoundObject::AutomationRemoteConnectionBoundObject(bytecode::OperandId operandId, AutomationRemoteOperation& parent)
```

Comment:
Final nit. To match the order in the header file, this should go before `AutomationRemoteAnyObject::AutomationRemoteAnyObject`